index.js

```diff
@@ -506,13 +506,16 @@ class GithubScm extends Scm {
         // For pull requests
         if (config.prRef) {
             const prRef = config.prRef.replace('merge', 'head:pr');
+            const prBranch = prRef.split(':')[0];
```

Comment:
Is this name of Branch?

Reply:
Yes!
The following value ​come in.
`prBranch = 'pull/1/head'`

Reply:
There is a problem with this method, so I fix.